lib/honeybadger/config/defaults.rb

```diff
@@ -300,6 +300,11 @@ class Boolean; end
         default: true,
         type: Boolean
       },
+      :'rails.subscriber_ignore_sources' => {
+        description: "Sources that should be ignored when using the Rails' (7+) native error reporter.",
+        default: [/\w+_cache_store.active_support/],
```

Comment:
Just curious, is there any addition context around why someone wouldn't want to see cache-related errors by default?

Reply:
I think they could easily pile up and use up their quota. Bt perhaps I should look more closely at when ActiveSupport specifically reports an error for those.

Reply:
Hmm, seems like a scenario where Reids errors are raised includes [when the cluster is unavailable](https://github.com/rails/rails/blob/95fa021465499c820f3b8758bb014b8549b2d13d/activesupport/test/cache/stores/redis_cache_store_test.rb#L350), plus a number of other scenarios you might want to know about, so perhaps I'll leave it un-ignored by default.

![image](https://user-images.githubusercontent.com/14361073/183775074-4fd5f74b-5c62-42cb-a917-c1eda5ece577.png)